samples/bluetooth/central_hr_coded/boards/nrf54h20dk_nrf54h20_cpuapp.conf

```diff
@@ -0,0 +1,8 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_BT_CTLR_ADV_EXT=n
+CONFIG_BT_CTLR_PHY_CODED=n
```

Comment:
Why is PHY_CODED disabled on hr_coded sample?

Reply:
Please note that application core doesn't have BLE controller so default configuration needs to be overwritten to not produce warnings. Coded PHY for controller is enabled on radio core. The same about below comments

Reply:
Please add comments in the files about this, especially about PHY_CODED because it might be misleading.

Reply:
I agree. Done